samples/java-eventsourced-counter/src/main/proto/com/example/kalix_policy.proto

```diff
@@ -0,0 +1,14 @@
+// This is the default Access Control List (ACL) for all components of this Kalix Service
+syntax = "proto3";
+
+package customer;
+
+import "kalix/annotations.proto";
+
+// Allow all other Kalix services deployed in the same project to access the components of this
+// Kalix service, but disallow access from the internet. This can be overridden explicitly
+// per component or method using annotations.
+// Documentation at https://docs.kalix.io/services/using-acls.html
+option (kalix.file).acl = {
+  allow: { service: "*" }
+};
```

Comment:
We'd likely also want to open up the services of each sample to internet, if we don't rather want to allow that as default. If not it means that building a sample and deploying it and then exposing means it won't be accessible, which is not super nice for someone trying Kalix out the first time.